### DISCUSSION TITLE
Please let us know your sig preferences and constraints by 2pm (local time)

### DISCUSSION THREAD

**Discussion by fcbond** [delph-in/docs#71] [Team Posts]
https://docs.google.com/spreadsheets/d/1K6qV1APItPu7edtKN6dUO81Vb86DaksAIZlQUfyyRlM/edit?usp=sharing

**Comment by arademaker** [comment]
Given that I can only make Friday, I will not be able to attend many SIGs and tutorials for sure.  I marked the ones I am interested hoping the recording of them can also be made available! 

Sent from my iPhone
